core/templates/dev/head/pages/exploration_editor/feedback_tab/ThreadDataService.js

```diff
@@ -51,7 +51,7 @@ oppia.factory('ThreadDataService', [
     var _openThreadsCount = 0;
 
     var _fetchThreads = function(successCallback) {
-      var fPromise = $http.get(_THREAD_LIST_HANDLER_URL);
+      var threadsPromise = $http.get(_THREAD_LIST_HANDLER_URL);
```

Comment:
If it's only the suggestion threads then perhaps call it suggestionThreadsPromise.

Reply:
It gets both feedback threads and suggestion threads. So I left the name as it is.